src/main/actions/file.js

```diff
@@ -26,19 +26,31 @@ const watchAndReload = (pathname, win) => { // when i build, and failed.
   // })
 }
 
-const writeFile = (pathname, content, extension, win, e) => {
+const writeFile = (pathname, content, extension, win, e, quitAfterSave = false) => {
   if (pathname) {
     pathname = pathname.endsWith(extension) ? pathname : `${pathname}${extension}`
-    const filename = path.basename(pathname)
-    fs.writeFile(pathname, content, 'utf-8', err => {
-      if (err) log(err)
+    writeFileToFilesystem(pathname, content, e, err => {
+      const filename = path.basename(pathname)
       // not export
       if (extension === '.md' && e) e.sender.send('AGANI::set-pathname', { pathname, filename })
+      if (!err && quitAfterSave) forceClose(win)
     })
     // watchAndReload(pathname, win)
   }
 }
 
+const writeFileToFilesystem = (pathname, content, e, callback = null) => {
+  fs.writeFile(pathname, content, 'utf-8', err => {
+    if (err) {
+      // TODO(fxha): Show user a/the error message
+      log(err)
+    } else {
+      e.sender.send('AGANI::file-saved-successfully')
```

Comment:
export `PDF` and `HTML` also use the function, I think only `save markdown to file` need to send `AGANI::file-saved-successfully` ? So is it better to put `AGANI::file-saved-successfully` in the callback?